pom.xml

```diff
@@ -66,6 +66,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.build.resourceEncoding>UTF-8</project.build.resourceEncoding>
 
+    <moditect.skip>true</moditect.skip>
```

Comment:
Can we make default false and only declare skip = true on modules that need it?

So most submodules will be unchanged

Reply:
Be aware that the root must skip otherwise it will trigger an error, that's why the default is set to `true`.